app/models/optimized_image.rb

```diff
@@ -24,7 +24,7 @@ def self.lock(upload_id, width, height)
   end
 
   def self.create_for(upload, width, height, opts = {})
-    return unless width > 0 && height > 0
+    return if width <= 0 || height <= 0
```

Comment:
Could also be `return if !(width > 0 && height > 0)`, but if you prefer the other way that is fine too 👍